transform/downscale_local_mean/downscale_local_mean.py

```diff
@@ -0,0 +1,180 @@
+import tensorflow as tf
```

Comment:
Always good to add a header block. Block would include authorship, license, copyright, dates and description of what the driver/module does.